src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -14,81 +16,125 @@
 import hashlib
 import traceback
 import re
+from functools import wraps
 
 try:
-    from azure.cli.core import __version__ as core_version
-except: #pylint: disable=bare-except
-    core_version = None
-try:
     from azure.cli.core._util import CLIError
-except: #pylint: disable=bare-except
+except:  # pylint: disable=bare-except
     CLIError = Exception
```

Comment:
Question, is this necessary? What's the historical context of this? Will this even happen?